### PR TITLE
For external Workshop

### PR DESCRIPTION
**Pull request by julieplummer20**

### CODE REVIEW

File: tutorials/abap-environment-rap-generator/abap-environment-rap-generator.md [hunk 1]
```diff
@@ -0,0 +1,207 @@
...
+auto_validation: true
+time: 10
+tags: [ tutorial>beginner, software-product>sap-btp--abap-environment]
+primary_tag: programming-tool>abap-development
+---
+
+## Prerequisites
+- You have done one of the following:
+    - **Tutorial**: [Create an SAP BTP ABAP Environment Trial User](abap-environment-trial-onboarding)
+    - You have bought a licensed version of SAP BTP ABAP Environment
```
Comment (MichaelCzcz) [inline tutorials/abap-environment-rap-generator/abap-environment-rap-generator.md:13]: Wrong indentation

File: tutorials/abap-environment-rap-generator/abap-environment-rap-generator.md [hunk 2]
```diff
@@ -0,0 +1,207 @@
...
+
+    !![step1a-new-package](step1a-new-package.png)
+
+2. Enter the following then follow the wizard, choosing a **new** transport request:
+- Name: **`ZXXX_TRAVEL`**
+- Description: **Travel data generated**
+- Request description: **Travel service RAP Generator**
+
+    !![step1a-create-package](step1a-create-package.png)
+    .
```
Comment (MichaelCzcz) [inline tutorials/abap-environment-rap-generator/abap-environment-rap-generator.md:39]: Please remove this

File: tutorials/abap-environment-rap-generator/abap-environment-rap-generator.md [hunk 3]
```diff
@@ -0,0 +1,207 @@
...
+
+    ```
+
+7. Format, save, and activate your table ( **`Shift+F1, Ctrl+S, Ctrl+F3`** ).
+
+8. Optional: Check your table by clicking in the editor and choosing **Open With > Data Preview**.
+
+    !![step2c-data-preview](step2c-data-preview.png)
+
+> You must include a key field `client` with the type `abap.clnt`.
```
Comment (MichaelCzcz) [inline tutorials/abap-environment-rap-generator/abap-environment-rap-generator.md:96]: Add indentation

File: tutorials/abap-environment-rap-generator/abap-environment-rap-generator.md [hunk 4]
```diff
@@ -0,0 +1,207 @@
...
+
+> You must include the following administrative fields with the following types.
```
Comment (MichaelCzcz) [inline tutorials/abap-environment-rap-generator/abap-environment-rap-generator.md:98]: Add indentation

File: tutorials/abap-environment-rap-generator/abap-environment-rap-generator.md [hunk 5]
```diff
@@ -0,0 +1,207 @@
...
+
+> These are known as reuse data elements.
```
Comment (MichaelCzcz) [inline tutorials/abap-environment-rap-generator/abap-environment-rap-generator.md:100]: Add indentation

File: tutorials/abap-environment-rap-generator/abap-environment-rap-generator.md [hunk 6]
```diff
@@ -0,0 +1,207 @@
...
+
+    ```ABAP
```
Comment (MichaelCzcz) [inline tutorials/abap-environment-rap-generator/abap-environment-rap-generator.md:102]: Add indentation for code block

**Review by MichaelCzcz — CHANGES_REQUESTED** [review]
Thank you for your PR. Please fix the formatting issues.

File: tutorials/abap-environment-rap-generator/abap-environment-rap-generator.md (hunk 1 above)
Comment (julieplummer20) [reply tutorials/abap-environment-rap-generator/abap-environment-rap-generator.md:13]: Hi Michael, This indentation is intentional. I discussed it with Daniel:
"- You have done one of the following:" is first level
    - **Tutorial**: [Create an SAP BTP ABAP Environment Trial User](abap-environment-trial-onboarding)
    - You have bought a licensed version of SAP BTP ABAP Environment
... are subordinate to "- You have done one of the following:" (ie they are second-level)
... whereas "- You have installed [ABAP Development Tools](https://tools.hana.ondemand.com/#abap), latest version" is first-level

**Pull request merged**